src/main/scala/com/ing/wbaa/gargoyle/proxy/data/S3Request.scala

```diff
@@ -21,8 +25,18 @@ object S3Request extends LazyLogging {
       .toList
       .lift(1)
 
-    val accessType = if (httpMethod == HttpMethods.GET) Read else Write
+    val bucketObject = path.toString
```

Comment:
The `.lift(2)` will now only get the first level of the object path.

Should we make this a bucketObjectPath perhaps? If we make that a List[String] we can immediately model the entire path instead of only the first level. Or alternatively we can call the current variable bucketObjectRoot or similar perhaps, to make clear it's only the first level